nucypher/characters/banners.py

```diff
@@ -76,4 +76,5 @@
 
 
 the Untrusted Re-Encryption Proxy.
+{}
```

Comment:
Sure, why not.

Reply:
I am sometimes filling this with the checksum address when it is available - it helps with log readability.

Reply:
Nickname and checksum address ideally, right?

Reply:
Yes, Ideally, sometimes we dont have it yet.

Reply:
:heavy_check_mark: